ydb/core/protos/feature_flags.proto

```diff
@@ -172,4 +172,5 @@ message TFeatureFlags {
     optional bool EnableInsertWriteIdSpecialColumnCompatibility = 147 [default = false];
     optional bool EnableTopicAutopartitioningForCDC = 148 [default = false];
     optional bool EnableWritePortionsOnInsert = 149 [default = false];
+    optional bool EnableFollowerStat = 150 [default = false];
```

Comment:
I would suggest to make it EnableFollowerStat**s** -- to be in line with `EnablePersistentPartitionStats`.

Reply:
Done.